src/resolve/resolved.conf.in

```diff
@@ -35,3 +35,4 @@
 #ReadEtcHosts=yes
 #ResolveUnicastSingleLabel=no
 #StaleRetentionSec=0
+#RefuseRecordTypes=AAAA,MX
```

Comment:
This must match the default, hence add the key without any value

Reply:
@bluca done